chatbot/chatbot_dependencies.py

```diff
@@ -0,0 +1,19 @@
+# Import Python Packages
```

Comment:
Why we need such a file?

Reply:
Emmm, I wrote this before because I was worried that there were too many packages to import, so considering the scalability, I want to list them separately. But now it seems that there is no need for this.
Anyway, I will delete it.